libwayshot/src/lib.rs

```diff
@@ -77,6 +77,7 @@ struct IntersectingOutput {
 pub struct WayshotConnection {
     pub conn: Connection,
     pub globals: GlobalList,
+    output_infos: Vec<OutputInfo>,
```

Comment:
Hm would we benefit from changing this to pub?

Reply:
I think it should not be changed by user..

Reply:
Ok that makes sense